client/starwhale/api/_impl/job.py

```diff
@@ -178,6 +179,8 @@ def parse_job_from_module(module: str, path: Path) -> t.Dict[str, t.List[t.Dict]
         Parser.set_parse_stage(True)
         # parse DAG
         logger.debug(f"parse @step for module:{module}")
+        if module in sys.modules:
```

Comment:
why delete the imported module?

If there are some reasons, the modules delete logic moves to the load_module function?

Reply:
#### why delete the imported module?

This function is used for parse @step when importing from module, but modules will not be loaded again if they have been loaded which will cause empty steps from job. Force loading modules makes sure `parse_job_from_module` get the desired steps in what ever case.

#### Why not in load_module function:

Force loading feature may not be needed by load_module function  .

Reply:
ok